category/basic.lean

```diff
@@ -95,6 +97,24 @@ by rw [← bind_pure_comp_eq_map, bind_assoc]; simp [pure_bind]
 lemma seq_eq_bind_map {x : m α} {f : m (α → β)} : f <*> x = (f >>= (<$> x)) :=
 (bind_map_eq_seq m f x).symm
 
+@[reducible]
+def pipe {m} [monad m] {α β γ} (f : α → m β) (g : β → m γ) := λ x, f x >>= g
+
+infix ` >=> `:55 := pipe
```

Comment:
This is already in the core library:
```lean
infix ` >=> `:2 := λ s t a, s a >>= t
```
But maybe you specifically want a named version?

Reply:
Do you think that a different name would be better than `pipe` in this regard?